pvlib/tests/test_modelchain.py

```diff
@@ -682,6 +713,22 @@ def test_deprecated_08():
                    temperature_model='pvsyst', temp_model='sapm')
 
```

Comment:
commenting here since it's the closest I can get to the relevant line... change the `ac_model` specs in `test_deprecated_08` so that new warnings are not emitted.